lib/serialDevice.js

```diff
@@ -17,6 +17,16 @@
 
 var _ = require('lodash');
```

Comment:
Looks like `import` was only done for the native dependencies to have them resolve with Babel instead of Webpack, see 0a0818a139. Not sure if this is the right time to ES6-ify the whole file. We probably need to come up with a migration strategy for which aspects we want to ES6-ify?

Reply:
Fair enough.